src/sc_linac_physics/applications/tuning/tuning_gui.py

```diff
@@ -155,12 +155,11 @@ def populate_detune_plot(self):
             detune_pvs.append(cavity.detune_best_pv)
             cold_pvs.append(cavity.df_cold_pv)
         colors = make_rainbow(len(detune_pvs) * 2)
```

Comment:
I think we can make this slightly simpler by removing the `* 2` here and then just using `idx ` further down